app/src/main/res/layout/fragment_explore.xml

```diff
@@ -20,31 +33,28 @@
             app:titleTextColor="@color/black">
 
             <!--<SearchView-->
-                <!--android:id="@+id/searchView"-->
-                <!--android:layout_width="wrap_content"-->
-                <!--android:layout_height="wrap_content"-->
-                <!--android:layout_gravity="end"-->
-                <!--android:queryHint="Search"-->
-                <!--android:layout_centerHorizontal="true" />-->
+            <!--android:id="@+id/searchView"-->
+            <!--android:layout_width="wrap_content"-->
+            <!--android:layout_height="wrap_content"-->
+            <!--android:layout_gravity="end"-->
+            <!--android:queryHint="Search"-->
+            <!--android:layout_centerHorizontal="true" />-->
 
         </androidx.appcompat.widget.Toolbar>
 
     </com.google.android.material.appbar.AppBarLayout>
+
     <com.google.android.material.tabs.TabLayout
         android:id="@+id/tabLayout"
         android:layout_width="match_parent"
         android:layout_height="?attr/actionBarSize"
-        android:layout_gravity="bottom"
-        app:tabTextAppearance="@android:style/TextAppearance.Widget.TabWidget"
         android:background="@color/colorPrimary"
-        app:tabSelectedTextColor="@color/colorAccent"
```

Comment:
@MarinaTaran we need to make sure the textColo,r is same. Can you add the below lines ? 

```
        app:tabTextAppearance="@android:style/TextAppearance.Widget.TabWidget"
        app:tabSelectedTextColor="@color/colorAccent"

```

Reply:
ok.I'll add it

Reply:
we have in fragment_explore this lines
![20191024_191947](https://user-images.githubusercontent.com/42735198/67505627-4da89f80-f694-11e9-9ad7-8afee433b03b.jpg)

Reply:
pleas,check "code", we have this lines